src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java

```diff
@@ -0,0 +1,33 @@
+package com.github.dockerjava.api.command;
+
+import com.github.dockerjava.api.model.Service;
+
+import javax.annotation.CheckForNull;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * List containers
+ *
+ */
+public interface ListServicesCmd extends SyncDockerCmd<List<Service>> {
+
+    @CheckForNull
+    Map<String, List<String>> getFilters();
+
+    /**
+     * @param ids
+     *            - Show only services with the given ids
+     */
+    ListServicesCmd withIdFilter(String... ids);
+    /**
+     *
+     * @param names
+     *            - Show only services with the given names
+     */
+    ListServicesCmd withNameFilter(String... names);
```

Comment:
> withNameFilters(Iterable<String>)?

`Iterable` has no `size` `isEmpty` that `Collection` has and pretty useful in end-user libraries. 
`List` allows do `addAll` and other useful operations. I.e. i'm using it when constructing  user configuration + system configuration for docker cmds.